product_assortment/models/ir_filters.py

```diff
@@ -40,7 +46,7 @@ def _get_default_is_assortment(self):
     def _get_eval_domain(self):
         res = super(ProductAssortment, self)._get_eval_domain()
 
-        if self.whitelist_product_ids and res:
+        if self.whitelist_product_ids:
```

Comment:
@CarlosRoca13 What's the behaviour of this code when filter is void and whitelist products filled in ?

Reply:
With this, we allow to show only the products set on whitelist. If we set this condition, we are going to see all the products when we set the domain as `[]` instead of see only the whitelisted.

Reply:
Makes sense ! Great

Reply:
@Cedric-Pigeon To be sure